media_driver/linux/common/codec/ddi/media_ddi_decode_vp8.cpp

```diff
@@ -264,21 +264,12 @@ VAStatus DdiDecodeVP8::ParseProbabilityData(
 {
     if (vp8ProbDataBuff->pData && probInputBuf)
     {
-        MOS_LINUX_BO *boDes = nullptr;
-        boDes               = vp8ProbDataBuff->bo;
-
-        mos_bo_wait_rendering(boDes);
-        if (0 == mos_bo_subdata(boDes,
-                     0,
-                     sizeof(CODECHAL_VP8_COEFFPROB_DATA),
-                     probInputBuf))
-        {
-            return VA_STATUS_SUCCESS;
-        }
-        else
-        {
-            return VA_STATUS_ERROR_INVALID_PARAMETER;
-        }
+        mos_bo_wait_rendering(vp8ProbDataBuff->bo);
```

Comment:
if there are no a bo_map , vp8ProbDataBuff->pData should be NULL, should we add some protection or just add bo_map before memory copy?

Reply:
This is already guarded by
```
    if (vp8ProbDataBuff->pData && probInputBuf)
```
if you look a couple lines up in the file.  It should be fine.

Reply:
yes, it already been checked, thanks